.github/workflows/build-release.yml

```diff
@@ -263,6 +263,20 @@ jobs:
           name: NuGet-Packages (${{ matrix.os }})
           path: './temp/output/Hazelcast.Net.*pkg'
 
+      # publish examples (inc. soak tests) (windows only so we have framework too)
+      - name: Publish examples
+        if: matrix.os == 'windows-latest'
+        shell: pwsh
+        run: ./hz.ps1 -noRestore -localRestore publish-examples
```

Comment:
As I understand for now this command is not yet present in the PR? 
`./hz.ps1 publish-examples` fails with `unknown command 'publish-examples' - use 'help' to list valid commands`

Reply:
but? should be part of the PR, fixing

Reply:
so, it disapeared in a poorly-controled git-rebase, now it's back - publishes to `temp/examples` - in theory you should be able to eg `cd temp/examples/netcoreapp3.1` and there, `hx.exe ...` should run the examples. also producing a zipped version so it's easy to deploy to, say, soak test machine.